tests/main_test.py

```diff
@@ -44,22 +63,18 @@ def test_json_output(self, mock_json, mock_find_vulnerabilities, mock_parse_args
 
         mock_discover_files.return_value = [example_file]
         mock_parse_args.return_value = mock.Mock(
-            autospec=True,
             project_root=None,
             baseline=None,
             json=True,
             output_file=output_file
         )
-        main([
-            'parse_args is mocked'
-        ])
+        with self.assertRaises(SystemExit):
+            main(['parse_args is mocked'])
         assert mock_json.report.call_count == 1
-        # This with: makes no sense
-        with self.assertRaises(AssertionError):
-            assert mock_json.report.assert_called_with(
-                mock_find_vulnerabilities.return_value,
-                mock_parse_args.return_value.output_file
-            )
+        mock_json.report.assert_called_with(
```

Comment:
You're my hero! :D
What was the bug? re: `This with: makes no sense` The `autospec=True`?

Reply:
@KevinHock no, the function `mock_text.report.assert_called_with()` does the assertion - it either raises `AssertionError` or returns `None`.

Therefore, if you do `assert mock_text.report.assert_called_with()` it's like `assert None`.

Reply:
Wow 🤦‍♂️ I must have had tunnel vision at the time. Thanks so much for fixing that 😄